doc/user_guide/marks/geoshape.rst

```diff
@@ -559,9 +566,11 @@ populous states. Using an ``alt.selection_point()`` we define a selection parame
 
 
 The interaction is two-directional. If you click (shift-click for multi-selection) on a geometry or bar the selection receive an ``opacity`` of ``1`` and the remaining an ``opacity`` of ``0.2``.
+It is also possible to create charts with interval selections,
+as can be seen in the :ref:`gallery_interval_selection_map_quakes` gallery example.
```

Comment:
This example is really cool, and a good place to redirect too for people who want to learn more